packages/odoo/test/Adaptor.test.js

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import {
+  execute,
+  create,
+  dataValue,
+  update,
+  read,
+  deleteRecord,
+} from '../src/Adaptor.js';
+
+import MockAgent from './mockAgent.js';
```

Comment:
We should be able to remove all this now, as we're not using the HTTP mock?

We should remove mockAgent.js entirely